lib/ask/runtime/nuntium_channel.ex

```diff
@@ -280,9 +281,10 @@ defmodule Ask.Runtime.NuntiumChannel do
 
       respondent = NuntiumChannel.update_stats(respondent)
 
-      Nuntium.Client.new(channel.base_url, channel.oauth_token)
+      response = Nuntium.Client.new(channel.base_url, channel.oauth_token)
       |> Nuntium.Client.send_ao(channel.settings["nuntium_account"], messages)
 
+      SurvedaMetrics.increment_counter_with_label(:surveda_nuntium_enqueue, [elem(response, 0)])
```

Comment:
This will label the status with "ok" or "error" instead of the HTTP status code from Nuntium